controls/radrichtextbox/features/selection.md

```diff
@@ -10,32 +10,106 @@ published: True
+
+
+#### [C#] Example 1: Customize the appearance of the selection
+{{region radrichtextbox-features-selection_6}}
+
+	this.radRichTextBox.SelectionFill = new System.Windows.Media.SolidColorBrush(Colors.Red); 
+	this.radRichTextBox.SelectionFill.Opacity = 0.2;
+	this.radRichTextBox.SelectionStroke = System.Windows.Media.Brushes.DarkRed;
+{{endregion}}
+
+
+#### [VB.NET] Example 1: Customize the appearance of the selection
+{{region radrichtextbox-features-selection_7}}
+	
+	Me.radRichTextBox.SelectionFill = New SolidColorBrush(Colors.Red)
+	Ме.radRichTextBox.SelectionFill.Opacity = 0.2
+	Ме.radRichTextBox.SelectionStroke = Brushes.DarkRed
+{{endregion}}
+
+#### Figure 1: Custom selection colors
+![Customized Selection in RadRichTextBox](images/RadRichTextBox_Selection_02.png)
+
 
 ## Programmatic Selection
 
-The developer is allowed to work with the selection programmatically. This can be used when implementing [Search functionality]({%slug radrichtextbox-features-search%}) functionality in your __RadRichTextBox__ and you want to select the found string. 
+The developer is allowed to manipulate the selection programmatically. This can be used in a lot of scenarios involving editing of the document, or for example, when implementing [Search functionality]({%slug radrichtextbox-features-search%}) in your **RadRichTextBox** and you want to select the found string. 
+
+The programmatic selection is represented via the **DocumentSelection** class. The instance of the class gets associated with the **RadDocument** of **RadRichTextBox** and allows you to specify selection start and end positions, selection ranges and other. You can manage the selection by either using the **Selection** property of **RadDocument** or by creating an instance of the **DocumentSelection** class.
+
+>tipTo learn more about the **DocumentPosition** read the [Positioning]({%slug radrichtextbox-features-positioning%}) topic.
+
+
+### Members of DocumentSelection
+
+Properties:
+
+* **Ranges**: Holds a collection of all [**SelectionRange**](#what-is-selection-range) instances in the selection.
+* **IsEmpty**: Holds a value determining whether the selection is empty or not.
+
+Methods for manipulating the selection:
+
+* **SelectAll()**: Selects all the content of the document.
+* **SetSelectionStart()**: Empties the selection and begins a new [**SelectionRange**](#what-is-selection-range) starting at the specified position. This method starts the creation of a range, which should be finished with **AddSelectionEnd()**.
+* **AddSelectionStart()**: Begins a new [**SelectionRange**](#what-is-selection-range) starting at the specified position. This method starts the creation of a range, which should be finished with **AddSelectionEnd()**.
+* **AddSelectionEnd()**: Finishes the started [**SelectionRange**](#what-is-selection-range) by setting its end.
+* **SelectAnnotationRange()**: Selects annotation range including the AnnotationRangeStart and AnnotationRangeEnd elements.

```

Comment:
Should we change this to "Selects annotation range including the AnnotationRangeStart, AnnotationRangeEnd, and all content between them"?